pkg/apis/admissionregistration/types.go

```diff
@@ -406,9 +417,9 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 
 	// ParamRef specifies the parameter resource used to configure the admission control policy.
 	// It should point to a resource of the type specified in ParamKind of the bound ValidatingAdmissionPolicy.
-	// If the policy specifies a ParamKind and the resource referred to by ParamRef does not exist, this binding is considered mis-configured and the FailurePolicy of the ValidatingAdmissionPolicy applied.
+	// This field is ignored if the ParamKind if the policy is not set.
```

Comment:
Rephrase? I wasn't able to understand what this is saying.

Reply:
made another attempt at phrasing, PTAL